### PR TITLE
M3 test runners

### PR DESCRIPTION
**Pull request by OleksandrMelnyk4**
image
image

### CODE REVIEW

File: src/test/java/tests/FiltersTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  void filtersPageShouldHaveItems() {
+    login.loginWithUser(ADMINISTRATOR);
+    navigationMenuComponent.navigateToMenu(FILTERS);
+    filtersPage.gridShouldContainsFilters();
+  }
+
+  @ParameterizedTest(name = "Verify filters grid contains {0} column")
+  @MethodSource("filterColumnsDataProvider")
+  @Description("Verify filters grid contains proper columns")
+  public void verifyFilterGridContainsProperColumns(final String column) {
```
Comment (elenatar) [inline src/test/java/tests/FiltersTest.java]: Access modifier public could be removed as in other tests

Comment (OleksandrMelnyk4) [reply src/test/java/tests/FiltersTest.java]: Done. Thnaks

**Review by vzprj — APPROVED** [review]

File: src/test/java/buisness/FiltersPage.java [hunk 2]
```diff
@@ -29,4 +36,21 @@ public void applySearchFilterByName(final String filterName) {
...
+    $x(FILTER_ITEM_LINE + FILTER_NAME_TEMPLATE.formatted(filterName))
+      .shouldBe(visible).click();
+  }
+
+  public void mainGridContainsColumns(final String columnName) {
+    $x(GRID_COLUMN_NAME_TEMPLATE.formatted(columnName)).shouldBe(visible);
+    log.info("Column name is present %s".formatted(columnName));
+  }
+
+  public void openAnyFilter() {
```
Comment (mobilman89) [inline src/test/java/buisness/FiltersPage.java]: I believe the better name is openFirstFilter

File: src/test/java/core/utils/services/DataDrivenProvider.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package core.utils.services;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Resources;
+import lombok.SneakyThrows;
+
+import java.util.List;
+
+public class DataDrivenProvider {
```
Comment (mobilman89) [inline src/test/java/core/utils/services/DataDrivenProvider.java]: Remove useless class

**Review by mobilman89 — COMMENTED** [review]
I don't see BaseTest for an alternative runner

File: src/test/java/tests/FiltersTest.java
Comment (mobilman89) [inline src/test/java/tests/FiltersTest.java:1]: I don't see 3rd parametrized test

Comment (mobilman89) [inline src/test/java/tests/FiltersTest.java:1]: I don't see implementation for alternative runner

**Review by mobilman89 — APPROVED** [review]

File: src/test/java/core/utils/services/DataDrivenProvider.java (hunk 3 above)
Comment (OleksandrMelnyk4) [reply src/test/java/core/utils/services/DataDrivenProvider.java]: done

**Pull request merged**